exasol/error/_parse.py

```diff
@@ -38,16 +40,17 @@ def __iter__(self) -> Generator[ast.Call, None, None]:
         )
 
 
-def _extract_parameters(node: ast.Call):
-    kwargs = {}
+def _extract_parameters(node: ast.Call) -> Tuple[Any, Any, Any, Any]:
```

Comment:
In theory, are the type more or less known if I look at https://github.com/exasol/error-reporting-python/blob/827a1ba5bf2f054d593d8f119ac1db775ae20247/exasol/error/_parse.py#L92

Reply:
I tried to use the ast types, but it's complicated

Reply:
```suggestion
def _extract_parameters(node: ast.Call) -> Tuple[ast.AST,ast.AST,ast.AST,ast.AST]:
```

Reply:
maybe use dataclass

Reply:
Done